src/Core/Users/user.php

```diff
@@ -0,0 +1,142 @@
+<?php
+namespace Meetup\Login\Core;
+class User implements UserInterface {
+    protected $firstName;
+    protected $lastName;
+    protected $emailid;
+    protected $created;
+    protected $datetime;
+    protected $passwd;
+    /**
+     * Set first name
+     *
+     * @param string $firstName
+     *
+     * @return void
+     */
+    public function setFirstName($firstName) {
+        $this->firstName = $firstName;
+    }
+	 /**
+     * Get first name
+     *
+     * @return string
+     */
+    public function getFirstName() {
+        return $this->firstName;
+    }
+    /**
+     * Set last name
+     *
+     * @param string $lastName
+     *
+     * @return void
+     */
+    public function setLastName($lastName) {
+        $this->lastName = $lastName;
+    }
+     /**
+     * Get last name
+     *
+     * @return string
+     */
+    public function getLastName() {
+        return $this->lastName;
+    }
+    /**
+     * Set email
+     *
+     * @param string $emailid
+     *
+     * @return void
+     */
+    public function setEmail($emailid) {
+        $this->email = $emailid;
```

Comment:
You declared a class variable `protected $emailid;` and then you are using ` $this->email ` . Does not make sense.